arm-runtimes/test-support/modify-compiler-rt-xml.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Helper script to modify the xml results from compiler-rt.
+
+# compiler-rt always puts all the test results into the "compiler-rt"
+# testsuite in the junit xml file. We have multiple variants of
+# compiler-rt, so the xml is modified to group the tests by variant.
+
+import argparse
+import os
+from xml.etree import ElementTree
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Reformat compiler-rt xml results")
+    parser.add_argument(
+        "--compiler-rt-build-dir",
```

Comment:
Small nit: I'm not a great fan of enormously long option names in general, but especially not if they're mandatory. If I _have_ to specify this option, can't we at least have a `-d` alias for it or something?

Reply:
I think this was a holdover from an earlier version that required multiple directories. There's no reason for the length now.